core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -4428,14 +4458,11 @@ components:
           description: |
             An explanation as to why the intent status is resolved as it is.
         committed_state_version:
-          type: integer
-          format: int64
-          minimum: 1
-          maximum: 100000000000000
+          $ref: "#/components/schemas/StateVersion"
```

Comment:
_(note from author)_

This is just my drive-by refactor. (i.e. introduce reusable `StateVersion` openapi type)